cdap-data-fabric/src/main/java/co/cask/cdap/data2/transaction/queue/hbase/HBaseQueueAdmin.java

```diff
@@ -236,7 +234,9 @@ public void drop(String name) throws Exception {
 
   @Override
   public void upgrade(String tableName, Properties properties) throws Exception {
-    AbstractHBaseDataSetAdmin dsAdmin = new DatasetAdmin(tableName, hConf, tableUtil, properties);
+    //TODO: check this
```

Comment:
What is this TODO about? If it is truly a TODO, there should be a JIRA associated with it.